docs/source/setup/ssh.rst

```diff
@@ -24,3 +24,50 @@ Or you can specify a hostfile that includes a list of hosts::
 The ``dask-ssh`` utility depends on the ``paramiko``::
 
     pip install paramiko
+
+
+CLI Options
+-------- 
+
+Launch a distributed cluster over SSH. A 'dask-scheduler' process will run
```

Comment:
You probably wanted to use double backticks here for code highlighting in rst format

    ``dask-scheduler``